src/utils/recordsCardUtils.js

```diff
@@ -1,28 +1,85 @@
 const recordsCardUtils = {
     methods: {
-        getButtonLabel(ItemTitle) {
-            let returnedTitle;
-            returnedTitle = ItemTitle;
-            switch (ItemTitle) {
-                case 'taxonomies':
-                    returnedTitle = 'species';
-                    break;
-                case 'userDefinedTags':
-                    returnedTitle = 'tags';
-                    break;
-                default:
-                    returnedTitle = ItemTitle;
-                    break;
-            }
-            return returnedTitle;
-        },
         getRecordLink(record) {
             if (record.doi) {
                 // Only the last part of the DOI is needed for local links.
                 return record.doi.split('/').pop();
             } else {
                 return record.id;
             }
+        },
+        getChipColor(chip){
+            if(chip.type==='subjects'){
+                return 'subject_color';
+            }
+            else if(chip.type==='domains')
+            {
```

Comment:
why do you put the { on the same line at line 12 and then go to new line at 16 and 20 ?

Reply:
thanks. it will be fixed